universal/pm.md

```diff
@@ -11,11 +11,7 @@ So before we can fine tune power management to our liking, we need to first make
 
 Apple dropped support for XCPM on these models back in macOS Sierra, so XCPM is only supported between 10.8.5 and 10.11.6. You will still need [ssdtPRgen](../universal/pm.md#sandy-and-ivy-bridge-power-management).
```

Comment:
Hm, is it actually worth mentioning that XCPM can still be enabled on Ivy Bridge on 10.12+ by patching XNU? If this might confuse users, then better not.

Reply:
probably will